src/types.ts

```diff
@@ -0,0 +1,14 @@
+export type bytes8 = Buffer;
+export type bytes32 = Buffer;
+export type bytes48 = Buffer;
+export type bytes96 = Buffer;
+
+export const bytes8 = "bytes8";
```

Comment:
I don't think we need these consts here.

Reply:
yeah probably, I will make some refactoring to name all the types, this was just assembled to see if I can pass spec tests. Now it's trivial to finish rest of the functions and clean up